cosmos/scripts/chaos/cosmosdb_client_network_chaos_tool/readme.md

```diff
@@ -0,0 +1,129 @@
+## Introduction to Cosmos DB client-side network chaos tool
```

Comment:
I am not sure what you meant by labeled as preview. Should I add some tag somewhere?

Reply:
Added 'Preview' in title

Reply:
I'm not sure what the guidelines are for labeling. I just wanted to avoid a customer raising an incident in the middle of the night if they a hit an issue.